src/shared/initial-params.ts

```diff
@@ -0,0 +1,4 @@
+export const initialParams = new URLSearchParams(window.location.href.split('?')[1]);
```

Comment:
Suggest `initialSearchParams`, as this language better suits the router (that uses the term `params` to mean url parms e.g. `:someId` is read from `useParams`